cypress-tests/cypress/e2e/configs/Payment/Novalnet.js

```diff
@@ -254,6 +254,17 @@ export const connectorDetails = {
         },
       },
     },
+    MITWithoutBillingAddress: {
+      Request: {
+        billing: null,
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "succeeded",
+        },
+      },
+    },
```

Comment:
i feel, it is better to be removed and just import from commons where ever it is same.

also, we can consider re-using it from configs.

what say?